appinventor/appengine/src/com/google/appinventor/client/wizards/FileUploadWizard.java

```diff
@@ -22,24 +22,42 @@
 import com.google.appinventor.shared.rpc.project.youngandroid.YoungAndroidAssetNode;
 import com.google.appinventor.shared.rpc.project.youngandroid.YoungAndroidAssetsFolder;
 import com.google.gwt.core.client.GWT;
-import com.google.gwt.user.client.Command;
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.uibinder.client.UiHandler;
 import com.google.gwt.user.client.Window;
-import com.google.gwt.user.client.ui.FileUpload;
-import com.google.gwt.user.client.ui.VerticalPanel;
+import com.google.gwt.user.client.ui.ClickListener;
 import com.google.gwt.user.client.ui.DialogBox;
-import com.google.gwt.user.client.ui.HTML;
+import com.google.gwt.user.client.ui.FileUpload;
 import com.google.gwt.user.client.ui.Button;
-import com.google.gwt.user.client.ui.ClickListener;
-import com.google.gwt.user.client.ui.Widget;
 import com.google.gwt.user.client.ui.FlowPanel;
+import com.google.gwt.user.client.ui.HTML;
+import com.google.gwt.user.client.ui.VerticalPanel;
+import com.google.gwt.user.client.ui.Widget;
+
 import java.util.Collection;
+import java.util.logging.Logger;
 
 
 /**
  * Wizard for uploading individual files.
  *
  */
-public class FileUploadWizard extends Wizard {
+public class FileUploadWizard {
+  interface FileUploadWizardUiBinder extends UiBinder<Dialog, FileUploadWizard> {}
+  private static final FileUploadWizard.FileUploadWizardUiBinder UI_BINDER = GWT.create(FileUploadWizard.FileUploadWizardUiBinder.class);
```

Comment:
I don't see why this is being qualified with the outer class name. It should be sufficient to just reference `FileUploadWizardUiBinder.class`.